api/cmd/api/http.go

```diff
@@ -75,6 +82,9 @@ func handleHTTPServer(ctx context.Context, u *url.URL, categoryEndpoints *catego
 	for _, m := range categoryServer.Mounts {
 		logger.Infof("HTTP %q mounted on %s %s", m.Method, m.Verb, m.Pattern)
 	}
+	for _, m := range resourceServer.Mounts {
```

Comment:
Is it possible to accept servers as an interface - `server` so that the code would read

```go

for _, s := range servers {
   for _, m := range s.Mounts() {
    logger.Info( .... ")  
  }
}
```

Reply:
not able to loop on `servers` not sure why but it is an array of `Server` interface